crates/ruff_linter/resources/test/fixtures/pep8_naming/N814.py

```diff
@@ -1,3 +1,8 @@
+# Error
```

Comment:
similarly here -- I understand why you added this comment, but it makes the diff in the snapshots quite hard to review

```suggestion
```